### PR TITLE
feat(TBD-12139): Support Synapse

### PR DESCRIPTION
**Pull request by sponomarova**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

**Review by AlixMetivier — APPROVED** [review]

File: main/plugins/org.talend.hadoop.distribution.synapse/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Manifest-Version: 1.0
+Service-Component: OSGI-INF/synapse.xml
+Bundle-ManifestVersion: 2
+Bundle-Name: Azure Synapse
+Bundle-SymbolicName: org.talend.hadoop.distribution.synapse;singleton:=true
+Bundle-Version: 7.3.1.qualifier
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.synapse/META-INF/MANIFEST.MF:6]: need to change those version to  8.0.1.qualifier

File: main/plugins/org.talend.hadoop.distribution.synapse/pom.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.talend.studio</groupId>
+    <artifactId>tbd-studio-se</artifactId>
+    <version>7.3.1-PATCH</version>
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.synapse/pom.xml:7]: need to change that version

File: test/plugins/org.talend.hadoop.distribution.synapse.test/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Azure Synapse Test
+Bundle-SymbolicName: org.talend.hadoop.distribution.synapse.test
+Bundle-Version: 7.3.1.qualifier
```
Comment (AlixMetivier) [inline test/plugins/org.talend.hadoop.distribution.synapse.test/META-INF/MANIFEST.MF:5]: need to change

File: test/plugins/org.talend.hadoop.distribution.synapse.test/pom.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.talend.studio</groupId>
+        <artifactId>tbd-studio-se</artifactId>
+        <version>7.3.1-PATCH</version>
```
Comment (AlixMetivier) [inline test/plugins/org.talend.hadoop.distribution.synapse.test/pom.xml:7]: need to be changed

**Review by AlixMetivier — CHANGES_REQUESTED** [review]

File: test/plugins/org.talend.hadoop.distribution.synapse.test/META-INF/MANIFEST.MF (hunk 3 above)
Comment (sponomarova) [reply test/plugins/org.talend.hadoop.distribution.synapse.test/META-INF/MANIFEST.MF]: Done

**Review by AlixMetivier — APPROVED** [review]

**Review by cbadillo1603 — APPROVED** [review]
reviewed

**Review by AlixMetivier — APPROVED** [review]

**Pull request merged**